### PR TITLE
add tensorflow-datasets, simclr

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rluria14** [comment]
@ndmaxar @oblute agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/tensorflow-datasets/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-datasets/meta.yaml:42]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/tensorflow-datasets/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-datasets/meta.yaml:27]: These and the other requirements may be able to be unconditional. Some of these packages were converted to empty but available packages on newer Python version. Removing the selectors would have the benefit this recipe being `noarch: python`. It will fail because some dependencies are not available on Windows but for `noarch: python` we are happy to merge if CI passes on Linux.

Comment (rluria14) [reply recipes/tensorflow-datasets/meta.yaml:27]: @xhochy thanks! I have made these updates

**Review by xhochy — APPROVED** [review]

File: recipes/tensorflow-datasets/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-datasets/meta.yaml:37]: Can you also add `tensorflow-metadata` to conda-forge? Seems like it isn't packaged yet. CI also suggests that this needs a stricter tensorflow pin. Windows is OK to fail but not with the current error ;)
```suggestion
    - tensorflow-metadata
    - tensforflow >=2.1
```

File: recipes/tensorflow-metadata/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-metadata/meta.yaml:21]:
```suggestion
    - python >=3.6,<4.0
```

File: recipes/tensorflow-metadata/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-metadata/meta.yaml:26]:
```suggestion
    - python >=3.6,<4.0
```

File: recipes/tensorflow-metadata/meta.yaml
Comment (xhochy) [inline recipes/tensorflow-metadata/meta.yaml:2]: Missing quotes break the CI job
```suggestion
{% set version = "0.30.0" %}
```

Comment (rluria14) [reply recipes/tensorflow-metadata/meta.yaml]: whoops. thanks.

**Comment by xhochy** [comment]
I think in the case of `tensorflow-metadata` it is probably easier to repackage the wheel. There are no binaries involved and dealing with `bazel` is quite a hassle.

**Comment by rluria14** [comment]
> I think in the case of `tensorflow-metadata` it is probably easier to repackage the wheel. There are no binaries involved and dealing with `bazel` is quite a hassle.

Yes indeed. I'd rather not mess with bazel. how can i repackage the wheel?

**Comment by xhochy** [comment]
Have a look at other tensorflow-* recipes: https://github.com/conda-forge/tensorflow-estimator-feedstock/blob/master/recipe/meta.yaml

**Comment by rluria14** [comment]
@xhochy `tensorflow-metadata` seems to be building now

**Pull request merged**
